docs/user-guide/basic-usage.md

```diff
@@ -1,5 +1,15 @@
 # Starting Styx
 
+## System Requirements
+
+Running Styx requires Java 1.8. It can be run with 1.8.0_45 or later versions. Earlier maintenance
+releases or Java 1.9 may work, but are not guaranteed to.
+
+## Downloading Styx from release
+
+In order to download styx from release go to [Styx releases](https://github.com/HotelsDotCom/styx/releases) and download the zip file for the version you want making sure you choose the file corresponding to whichever operating system you are using.
```

Comment:
I would add some commas to make it a little more readable (also capitalise Styx):

> In order to download Styx from release, go to [Styx releases](https://github.com/HotelsDotCom/styx/releases) and download the zip file for the version you want, making sure you choose the file corresponding to whichever operating system you are using.
Alternatively, you can build styx from source code.